### PR TITLE
Feature: expose getResponse

### LINKED ISSUE #133: add getResponse method
I am using this module with Nextjs (ssr) and it is the only one that works well. Thanks for that!

I'm combining it with Formik, and I'm needing the `grecaptcha.getResponse()` method, because the `onVerify` callback does not fit my need.

Is there any technical reason why it is not implemented?

### PR DESCRIPTION
**Pull request by AndyOGo**
fixes #133

### CODE REVIEW

**Comment by AndyOGo** [comment]
@sarneeh 
Alright I fixed the `eslint` errors inside CI/CD, but now *lcov* is broken fro coverage reports 🤔 
https://travis-ci.com/sarneeh/reaptcha/builds/132190924#L335-L345

**Comment by AndyOGo** [comment]
@sarneeh @tojacob
I would be glad if you can review this.

File: package.json [hunk 1]
```diff
@@ -42,6 +42,8 @@
     "nyc": "^13.2.0",
     "prettier": "^1.13.7",
     "pretty-quick": "^1.6.0",
+    "react": "^16.10.2",
```
Comment (sarneeh) [inline package.json]: I don't think these 2 are needed here.

File: package.json [hunk 2]
```diff
@@ -10,8 +10,8 @@
     "flow": "flow",
     "lint": "eslint .",
     "test": "ava --verbose 'lib/**/*.test.js'",
-    "coverage": "nyc yarn test && nyc report --reporter=text-lcov | coveralls",
-    "release": "yarn build && lerna publish --conventional-commits"
+    "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
```
Comment (sarneeh) [inline package.json]: Wow, I was just trying to reproduce the problem and it was a really tricky bug :D Thanks for resolving this!

File: package.json [hunk 3]
```diff
@@ -10,8 +10,8 @@
...
+    "release": "npm build && lerna publish --conventional-commits"
```
Comment (sarneeh) [inline package.json]: Shouldn't it be `npm run build`?

File: package.json (hunk 1 above)
Comment (sarneeh) [reply package.json]: I've cloned the repo, installed dependencies and tests were working fine without those two.

File: package.json (hunk 3 above)
Comment (AndyOGo) [reply package.json]: Thanks, will fix it

File: package.json (hunk 1 above)
Comment (AndyOGo) [reply package.json]: Interesting on my mac tested failed without `react` and `react-dom` 😕 
I will try again, first uninstall and if I still observe it, I will share logs and a screenshot, else I will undo it.

BTW. you are also sure, that you don't have `react` nor `react-dom` installed globally?

Comment (AndyOGo) [reply package.json]: Okay, I revert above and tried running tests locally, which throws `Error: Cannot find module 'react'` and broke test execution for me, by:

1. `npm uninstall --save-dev react react-dom`
2 `npm run test`

Screenshot 2019-10-16 at 21 23 49

```sh
npm run test

> reaptcha-monorepo@ test /Users/axawinterthur/dev/reaptcha
> ava --verbose 'lib/**/*.test.js'


  Uncaught exception in lib/index.test.js

  /Users/axawinterthur/dev/reaptcha/lib/index.test.js:3

   2: import test from 'ava';
   3: import sinon from 'sinon';
   4: import jsdom from 'jsdom-global';

  Error: Cannot find module 'react'

  Object.require (lib/index.test.js:3:37)
  Module._compile (node_modules/pirates/lib/index.js:99:24)
  Object.newLoader [as .js] (node_modules/pirates/lib/index.js:104:7)

  ✖ lib/index.test.js exited with a non-zero exit code: 1
  ✔ utils › isAnyScriptPresent › should return true
  ✔ utils › isAnyScriptPresent › should return false
  ✔ utils › injectScript › should inject script
  ✔ utils › injectScript › should not throw when document head not present

  4 tests passed
  1 uncaught exception

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! reaptcha-monorepo@ test: `ava --verbose 'lib/**/*.test.js'`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the reaptcha-monorepo@ test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/axawinterthur/.npm/_logs/2019-10-16T19_23_21_536Z-debug.log
```

**Comment by AndyOGo** [comment]
@sarneeh Thanks for your good and quick review.
I really like the code base and project setup 👍

File: package.json (hunk 1 above)
Comment (sarneeh) [reply package.json]: You shouldn't use `npm` on a `yarn`-based project. There's a `yarn.lock` file which defines what should be installed. I think that `react` and `react-dom` are just dependencies of react testing tools, so you don't need to install it manually.

Btw. by doing `npm uninstall react react-dom` you uninstall it from `package.json` AND uninstall all existing versions from `node_modules`. Try deleting `node_modules`, do an `yarn install` and run the tests. I bet it works!

Comment (AndyOGo) [reply package.json]: Alright. Thanks  for clarification. I will do that. Just give me sec.

Comment (AndyOGo) [reply package.json]: Thanks, you are right 👍 
If I understand it correctly, it seems that `yarn` deals with unmet peer dependencies automatically, in contrast to `npm` 🤔 
```sh
warning " > enzyme-adapter-react-16@1.11.2" has unmet peer dependency "react@^16.0.0-0".
warning " > enzyme-adapter-react-16@1.11.2" has unmet peer dependency "react-dom@^16.0.0-0".
warning "enzyme-adapter-react-16 > enzyme-adapter-utils@1.10.1" has unmet peer dependency "react@0.13.x || 0.14.x || ^15.0.0-0 || ^16.0.0-0".
warning "enzyme-adapter-react-16 > react-test-renderer@16.4.2" has unmet peer dependency "react@^16.0.0".
warning " > react-hot-loader@4.3.5" has unmet peer dependency "react@^15.0.0 || ^16.0.0".
warning " > reaptcha@1.6.0" has unmet peer dependency "react@^16.6.3".
```

Comment (AndyOGo) [reply package.json]: I reverted that commit 161419affb123b97401e255d78aeeaee49b088db

**Comment by sarneeh** [comment]
Thank you for your PR! Released in `1.7.0`.

**Comment by AndyOGo** [comment]
@sarneeh 
Sorry I just realized, I forgot to update the readme and maybe other related docs

**Comment by sarneeh** [comment]
@AndyOGo Oh, that's true. No problem. I've added a new issue for that so we won't forget: #185

**Comment by alfonmga** [comment]
I'm unable to use this method, did you guys forget to add types for TS? 

https://github.com/sarneeh/reaptcha/blob/master/lib/index.d.ts#L31

Screen Shot 2019-12-28 at 6 48 55 PM

**Comment by AndyOGo** [comment]
@alfonmga 
hmh, maybe I miss something 🤔 
But i thoought  `getResponse` is at this line, not?
https://github.com/sarneeh/reaptcha/blob/c0793aa83b0ec3e580d1b2c259e69836c408083d/lib/index.d.ts#L7

**Comment by alfonmga** [comment]
@AndyOGo I think that's fine, maybe @sarneeh can help here. Anyways we fixed that issue with PR #191. So don't worry it's all OK atm.

**Pull request merged**